src/Internal/Cardano/Types/Transaction.purs

```diff
@@ -312,10 +312,10 @@ instance Monoid TxBody where
     }
 
 instance EncodeAeson TxBody where
-  encodeAeson' (TxBody r) = encodeAeson' $ r
-    { inputs = encodeSet r.inputs
+  encodeAeson (TxBody r) = encodeAeson $ r
+    { inputs = r.inputs
```

Comment:
This is update, so I think there is no need to update the fields with same value.

Reply:
Also I do not understand why we do not need encodeSet anymore. What changed in that regard?

Reply:
We do not need it anymore, as now we have an EncodeAeson instance for Set.

Reply:
Indeed.